modules/steam.nix

```diff
@@ -409,5 +438,71 @@ in
         WINE_CPU_TOPOLOGY = "8:0,1,2,3,4,5,6,7";
       };
     })
+    (mkIf cfg.autoStart {
+      assertions = [
+        {
+          assertion = !config.systemd.services.display-manager.enable;
+          message = "No Display Managers must be enabled when jovian.steam.autoStart is used";
+        }
+      ];
+
+      services.xserver = {
+        enable = true;
+        displayManager.lightdm.enable = false;
+        displayManager.startx.enable = true;
+      };
+
+      systemd.services.gamescope-switcher = {
```

Comment:
In my WIP stuff, I had to add this to prevent it spuriously being restarted since I wanted to use VT1:

```
# This prevents nixos-rebuild from killing our app by activating getty again
systemd.services."autovt@tty1".enable = false;
```

I haven't tried this change set yet, but the issue it solves is that on system activation it ended-up being killed. note that I also used `conflicts = [ "getty@tty1.service" ];`...

Is using tty7 "safer" for that since there's no autovt going up that far?

Reply:
`greetd.service` has the correct `Conflicts=getty@tty1.service` so this shouldn't be an issue anymore.